web/src/main/java/no/nav/foreldrepenger/tilbakekreving/web/app/tjenester/forvaltning/ForvaltningBehandlingRestTjeneste.java

```diff
@@ -253,6 +253,26 @@ public Response korrigerHenvisning(@Valid @NotNull KorrigertHenvisningDto korrig
         return Response.ok().build();
     }
 
+    @POST
+    @Path("/sjekk-halvt-rettsgebyr")
+    @Operation(
+        tags = "FORVALTNING-behandling",
+        description = "Tjeneste for å hoppe tilbake dersom kan behandles automatisk!",
+        responses = {
+            @ApiResponse(responseCode = "200", description = "Aksjonspunkt avbrut"),
+            @ApiResponse(responseCode = "500", description = "ukjent feil.")
+        })
+    @BeskyttetRessurs(actionType = ActionType.CREATE, property = AbacProperty.DRIFT)
+    public Response sjekkMuligAutomatiskBehandling() {
+        behandlingRepository.finnBehandlingerFaktaFeilutbetaling().forEach(b -> {
```

Comment:
Burde man hente siste kravgrunnlag for å sjekke om det har vært en oppdatering, eller er ikke det relevant her?

Reply:
Og hadde det vært en ide og plukke ut en for å verifisere at fungerer ok?

Reply:
Her legger vi bare behandlinger på vent dersom de kan plukkes av AutomatiskSaksbehandlingBatchTask etter 8 uker.
Dersom det kommer et kravgrunnlag eller statusendring i mellomtiden så blir ting re-evaluert og det blir ny ventefrist dersom det fortsatt er lavt beløp.